apiary.apib

```diff
@@ -1142,7 +1142,13 @@ Get details for a single invoice.
                         + rate: 0.21 (number)
                         + taxable (Money)
                         + tax (Money)
+                + total_due: 0 (number)
```

Comment:
Not a big fan of this naming. Would rather have this just `due`.

Reply:
will change this

Reply:
I guess this change will also need to be reflected in the end-to-end tests?

Reply:
done

 :ok: by @luisabritotl on 31/01/2018 14:44:49